skietbaan_fe/src/bootstrap/LeaderboardStyle.css

```diff
@@ -2,52 +2,99 @@
     min-width: 100%;
 }
    
-.filterNav{
-    min-width: 100%;
-    display: inline-flex;
-    background-color: darkgrey;
-    padding: 1%;
+.filterSection{
+    background-color: #171F32;
+    color: white;
+    width: 100%;
 }
-.competitionSelction{
+.filterTableAlwaysVisible{
     width:100%;
-    background-color: black;
+    margin:3%;
+}
+.filterContent{
+    background-color: #171F32;
+    color: white;
+    width: 100%;
+    padding-left: 3%;
+    padding-right: 3%;
+    align-content: center;
+}
+.row div p{
+    color: #FF611D;
+}
+.row div .scrollableContainerC{
+    overflow-y: auto;
+    height: 150px;
+}
+.row .table td{
+  border: 0px;
+}
+.row div .scrollableContainerG{
+    overflow-y: auto;
+    height: 150px;
+}
+.container h4{
+    color: #FF611D;
+}
+.filterContent hr{
+    color: #FF611D;
+    background-color: #FF611D;
+}
+.row .table thead{
+    padding-left: 2%;
+    background-color: #171F32;
+    color: white;
+}
+.row .table td{
+    padding-right: 3%;
+}
+.row .table{
     padding: 2%;
 }
-.competitionSelction div{
-    margin: auto;
+.navbar{
+     background-color:white important; 
+   /*  color: white !important; */
 }
-.curentMember{
-    background-color: black;
+.nav-link{
     color: white;
-    position: fixed;
+}
+.firstcoll{
+   text-align: center
+}
+.curentMember{
+    background-color: #171F32 !important;
+    color: white !important;
+    position: fixed; 
     bottom: 0;
     left: 0;
     width: 100%;
+    padding: 0%;
+    padding-bottom: 2%;
 }
-.filterNav .col-md-4 .row{
-    display: -webkit-inline-box;
+.filters{
+    background-image: linear-gradient(-90deg, #FF611D, #171F32);
 }
-.groupFilt{
-    width:100%;
-    margin: 1%;
-    display: inline-block;
+.body{
+    background-color: #171F32 !important;
+    width: 100%;
 }
-.groupLabel{
-    text-align: center;
+.row{
+    width: 100%;
+    margin: 0% !important;
```

Comment:
!important?

Reply:
I am sure there is a boodstrap class to remove margin, and why are you using boodstrap rows if you anyway removing it's purpose

Reply:
m unisng it ,but for that specific , i want something of my own